public/components/FeatureAnywhereContextMenu/CreateAnomalyDetector/AddAnomalyDetector.tsx

```diff
@@ -157,11 +158,22 @@ function AddAnomalyDetector({
 
   const notifications = getNotifications();
   const handleValidationAndSubmit = (formikProps) => {
-    if (!isEmpty(formikProps.errors)) {
-      focusOnFirstWrongFeature(formikProps.errors, formikProps.setFieldTouched);
-      notifications.toasts.addDanger('One or more input fields is invalid');
+    if (formikProps.values.featureList.length !== 0) {
```

Comment:
What happens if it equals to 0? I think detector will still be created but it will fail to start, this isn't optimal, we should probably block creation here or not allow to delete if there is only one which is what we do in plugin page

Reply:
@amitgalitz The else block on line 173 handles the case if it equals 0 - will show the toast and block the creation

Reply:
oh total miss.. nvm